src/Azs.Network.Admin/test/Common.ps1

```diff
@@ -15,11 +15,14 @@
 # Tests to skip
 $global:SkippedTests = $(
     'TestGetAllVirtualNetworksOData',
-    'TestGetAllPublicIpAddressesOData'
+    'TestGetAllPublicIpAddressesOData',
```

Comment:
Why are these tests skipped? Please create issues and assign to network team.

Reply:
We can make a new task to go over all skipped tests. For now I think we should just commit working pipeline tests.

Reply:
Can you pls explain the reason for skipping? we can involve the network team and get them fixed

Reply:
Emailed Joshua Ya about this, CCed reviewers.